%-----------------------------------------------------------------------------------------------------------------------------------------------%
%	The MIT License (MIT)
%
%	Copyright (c) 2021 Jitin Nair
%
%	Permission is hereby granted, free of charge, to any person obtaining a copy
%	of this software and associated documentation files (the "Software"), to deal
%	in the Software without restriction, including without limitation the rights
%	to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
%	copies of the Software, and to permit persons to whom the Software is
%	furnished to do so, subject to the following conditions:
%	
%	THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
%	IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
%	FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
%	AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
%	LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
%	OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
%	THE SOFTWARE.
%	
%
%-----------------------------------------------------------------------------------------------------------------------------------------------%

%----------------------------------------------------------------------------------------
%	DOCUMENT DEFINITION
%----------------------------------------------------------------------------------------

% article class because we want to fully customize the page and not use a cv template
\documentclass[a4paper,12pt]{article}

%----------------------------------------------------------------------------------------
%	FONT
%----------------------------------------------------------------------------------------

% % fontspec allows you to use TTF/OTF fonts directly
% \usepackage{fontspec}
% \defaultfontfeatures{Ligatures=TeX}

% % modified for ShareLaTeX use
% \setmainfont[
% SmallCapsFont = Fontin-SmallCaps.otf,
% BoldFont = Fontin-Bold.otf,
% ItalicFont = Fontin-Italic.otf
% ]
% {Fontin.otf}

%----------------------------------------------------------------------------------------
%	PACKAGES
%----------------------------------------------------------------------------------------
\usepackage{url}
\usepackage{parskip} 	

%other packages for formatting
\RequirePackage{color}
\RequirePackage{graphicx}
\usepackage[usenames,dvipsnames]{xcolor}
% \usepackage[scale=0.9]{geometry}
\usepackage[scale=0.9]{geometry}
% \usepackage[
%     top=0.4in,
%     bottom=0.4in,
%     left=0.5in,
%     right=0.5in
% ]{geometry}

%tabularx environment
\usepackage{tabularx}

%for lists within experience section
\usepackage{enumitem}

% centered version of 'X' col. type
\newcolumntype{C}{>{\centering\arraybackslash}X} 

%to prevent spillover of tabular into next pages
\usepackage{supertabular}
\usepackage{tabularx}
\newlength{\fullcollw}
\setlength{\fullcollw}{0.47\textwidth}

%custom \section
\usepackage{titlesec}				
\usepackage{multicol}
\usepackage{multirow}

%CV Sections inspired by: 
%http://stefano.italians.nl/archives/26
\titleformat{\section}{\Large\scshape\raggedright}{}{0em}{}[\titlerule]
\titlespacing{\section}{0pt}{5pt}{10pt}

%for publications
% \usepackage[style=authoryear,sorting=ynt, maxbibnames=5, maxnames=5]{biblatex}
% \usepackage[
%     backend=biber,
%     style=authoryear,
%     sorting=ynt,
%     maxbibnames=5,
%     maxnames=5
% ]{biblatex}
\usepackage[
	backend=biber,
	style=authoryear,
	sorting=ydnt,
	giveninits=true,
	maxnames=6, minnames=6,
	maxbibnames=6, minbibnames=6,
	doi=false, isbn=false, url=false,
	date=year,
	dashed=false
]{biblatex}
\addbibresource{selected_citations.bib}
\addbibresource{patents.bib}

\defbibenvironment{bibliography}
  {\list{}{\leftmargin=1.5em\itemindent=0pt\itemsep=0.1em\parsep=0pt}}
  {\endlist}
  {\item[{\tiny$\bullet$}\hspace{0.2em}]}


%Setup hyperref package, and colours for links
\usepackage[unicode, draft=false]{hyperref}
\definecolor{linkcolour}{rgb}{0,0.2,0.6}
\hypersetup{colorlinks,breaklinks,urlcolor=linkcolour,linkcolor=linkcolour}
\addbibresource{citations.bib}
\setlength\bibitemsep{1em}

%for social icons
\usepackage{fontawesome5}

%debug page outer frames
%\usepackage{showframe}


% job listing environments
\newenvironment{jobshort}[2]
    {
    \begin{tabularx}{\linewidth}{@{}l X r@{}}
    \textbf{#1} & \hfill &  #2 \\[3.75pt]
    \end{tabularx}
    }
    {
    }

\newenvironment{joblong}[3]
    {
    \begin{tabularx}{\linewidth}{@{}l X r@{}}
    \textbf{#1} & \hfill &  #2 \\
    \textit{#3} & & \\[3.75pt]  
    \end{tabularx}
    \begin{minipage}[t]{\linewidth}
    \begin{itemize}[nosep,after=\strut, leftmargin=1em, itemsep=3pt,label=\scriptsize$\bullet$]
    }
    {
    \end{itemize}
    \end{minipage}    
    }

\newenvironment{projectlong}[2]
    {
    \begin{tabularx}{\linewidth}{@{}l X r@{}}
    \textbf{#1} & \hfill &  #2 \\
    \end{tabularx}
    \begin{minipage}[t]{\linewidth}
    \begin{itemize}[nosep,after=\strut, leftmargin=1em, itemsep=3pt,label=\scriptsize$\bullet$]
    }
    {
    \end{itemize}
    \end{minipage}    
    }
%----------------------------------------------------------------------------------------
%	BEGIN DOCUMENT
%----------------------------------------------------------------------------------------
\begin{document}

% non-numbered pages
\pagestyle{empty} 

%----------------------------------------------------------------------------------------
%	TITLE
%----------------------------------------------------------------------------------------

% \begin{tabularx}{\linewidth}{ @{}X X@{} }
% \huge{Your Name}\vspace{2pt} & \hfill \emoji{incoming-envelope} email@email.com \\
% \raisebox{-0.05\height}\faGithub\ username \ | \
% \raisebox{-0.00\height}\faLinkedin\ username \ | \ \raisebox{-0.05\height}\faGlobe \ mysite.com  & \hfill \emoji{calling} number
% \end{tabularx}

\begin{tabularx}{\linewidth}{@{} C @{}}
\Huge{Vedrana Ivezić} \\[7.5pt]
\href{https://github.com/ivezicV}{\raisebox{-0.05\height}\faGithub\ ivezicV} \ $|$ \ 
\href{https://linkedin.com/in/vedrana-ivezic-180021174/}{\raisebox{-0.05\height}\faLinkedin\ Vedrana Ivezic} \ $|$ \ 
\href{https://ivezicv.github.io/}{\raisebox{-0.05\height}\faGlobe \ ivezicv} \ $|$ \ 
\href{mailto:vivezic@ucla.edu}{\raisebox{-0.05\height}\faEnvelope \ vivezic@ucla.edu} \ 
% $|$ \ 
% \href{tel:+000000000000}{\raisebox{-0.05\height}\faMobile \ +00.00.000.000} \\
\end{tabularx}



%----------------------------------------------------------------------------------------
% EXPERIENCE SECTIONS
%----------------------------------------------------------------------------------------

Ph.D student in Medical Informatics at UCLA with experience developing representation learning models for multimodal biomedical data, including cytology, ultrasound, and wearables. Demonstrated expertise in Python, PyTorch, and large-scale self-supervised learning. Proven track record in building scalable AI systems for healthcare applications, with a focus on improving model generalization and robustness in low-resource clinical settings. Proven track record of publications and workshop papers in top venues such as CVPR, MICCAI, and JAMIA.
%Interests/ Keywords/ Summary
% \section{Summary}
% Deep learning researcher pursuing a PhD in Medical Informatics with experience in deep learning applications to medicine, particularly in wearable sensor data and imaging based cancer research. Interested in developing multi-modal and foundational models with a strong interest in self-supervised learning for representation learning across diverse biomedical data, including time series and large medical images. Proven track record of publishing in reputable conferences and journals.


% Deep learning researcher with a strong foundation in computer science and a track record of publishing in top conferences. Experienced in applying AI to wearable data and cancer research, with expertise in computer vision, multi-modal learning, and self-supervised methods. Passionate about advancing digital health and precision medicine through scalable, generalizable machine learning solutions, and eager to contribute to innovative projects at the intersection of healthcare and AI.

% Motivated researcher with experience applying deep learning to healthcare, particularly in wearable sensor data and cancer research. Skilled in developing and evaluating computer vision and multi-modal models, with a strong interest in self-supervised learning for representation learning across diverse biomedical data. Passionate about leveraging machine learning to advance digital health and precision medicine, with a focus on scalable methods that generalize across modalities and domains.

% Passionate and driven deep learning researcher with experience applying AI to healthcare challenges in wearable technology and cancer research. Strong interest in multi-modal learning, computer vision, and self-supervised methods, with a focus on developing innovative approaches that improve health outcomes and advance precision medicine. Eager to contribute to impactful projects at the intersection of technology and healthcare through an internship in deep learning.

% Deep learning researcher with experience applying AI to wearable data and cancer research, combining technical expertise in computer vision, multi-modal learning, and self-supervised methods with a passion for advancing digital health and precision medicine. Skilled at developing and evaluating models that generalize across data types, and eager to contribute to innovative projects at the intersection of healthcare and machine learning.

%----------------------------------------------------------------------------------------
%	EDUCATION
%----------------------------------------------------------------------------------------
\section{Education}
\begin{tabularx}{\linewidth}{@{}l X@{}}	
\textbf{University of California, Los Angeles, CA} &\hfill \normalsize Sept. 2022 - May 2027 \\
PhD student in Medical Informatics&\hfill \normalsize (\textit{expected)} \\

\textbf{Princeton University, Princeton, NJ} &\hfill Sept. 2018 - May 2022 \\ 

B.A. Computer Science& \hfill  \textit{GPA 3.74} \\

 Minor in Quantitative and Computational Biology  & \hfill  \textit{Magna cum laude} \\
\end{tabularx}



%----------------------------------------------------------------------------------------
%	Publications/Patents
%----------------------------------------------------------------------------------------
\section{Publications}
\begin{refsection}[selected_citations.bib]
\nocite{*}
\printbibliography[heading=none]
\end{refsection}
\textit{*Co-first authors. Please see more on \href{https://scholar.google.com/citations?user=xUyQJkcAAAAJ&hl=en}{Google Scholar}.}

%----------------------------------------------------------------------------------------
%	Experience
%----------------------------------------------------------------------------------------


%Experience
\section{Experience}

\begin{joblong}{Medical Informatics PhD Student - UCLA}{Sept 2022 - Present}{Biomedical AI Research Lab, University of California, Los Angeles}
\item Developing an automated thyroid cancer diagnostic pipeline  using radiology and cytology image analysis
\item Improving thyroid cancer risk stratification from cytological biopsy images with deep learning techniques
\item Building a Fitbit foundation model by integrating temporal patterns with large language models
\item Performing statistical analysis and predictive modeling on a private heart failure Fitbit dataset
% \item Developing an automated assistance pipeline for thyroid cancer diagnosis 
% \item Improving thyroid cancer risk stratification of cytological biopsy images using deep learning methods
% \item Developing a foundational model for Fitbit data using large language models 
% \item Statistics and predictive modeling for an in-house heart failure Fitbit dataset
\end{joblong}

\begin{joblong}{Undergraduate Researcher}{Sept. 2020 - May 20222}{Troyanskaya Lab, Princeton University}
\item Developed cell type–specific functional gene networks through Bayesian data integration to enhance gene prediction accuracy
\item Leveraged machine learning to prioritize candidate genes relevant to kidney disease mechanisms
\item Benchmarked cell type prediction methods across single-cell and single-nucleus RNA-seq datasets 
\item Analyzed differential gene expression to identify molecular changes across diseased kidney cell types
% \item Built cell type specific functional networks using Bayesian integration for gene prediction
% \item Leveraged the networks and machine learning techniques to predict kidney disease relevant genes
% \item Evaluated machine learning methods for cell type prediction in single cell and single nucleus sequencing data
% \item Examined differential gene expression data across diseased kidney cells using Seurat
\end{joblong}



\begin{joblong}{Research Intern}{June 2021 - April 2022}{Jarosz Lab, Stanford University}
\item Analyzed FASTQ sequencing data from yeast strains to identify gene essentiality and construct gene interaction networks
\item Applied statistical and computational methods to evaluate gene correlations associated with prion phenotypes
% \item Analyzed FASTQ data from yeast strains to identify gene essentiality and generate gene networks
% \item Employed statistical methods to evaluate gene correlations with prion phenotypes
\end{joblong}

\begin{joblong}{Research Intern}{June 2019 - Aug. 2019}{Tolić Group, Insitute Ruder Bošković, Zagreb, Croatia}
\item Imaged and analyzed mitotic spindles during pro-metaphase and metaphase to study spindle dynamics.
% \item Examined and videoed mitotic spindles in pro-metaphase and metaphase
\item Utilized transfection, cell splitting, and protein tagging methods to analyze PRC1 in pro-metaphase
\end{joblong}

\begin{joblong}{Research Intern}{June 2017 - Aug. 2017}{Intellectual Ventures Laboratory - Bellevue, WA}
\item Co-Inventor: “Methods and System for Concentration of Samples for Lateral Flow Assays”
\item Developed diagnostic assays to enable rapid and accurate identification of tuberculosis infection
% \item Helped develop new diagnostics to accurately and rapidly identify tuberculosis infection
\end{joblong}


%----------------------------------------------------------------------------------------
%	PROJECTS
%----------------------------------------------------------------------------------------

\section{Projects}

% \begin{projectlong}{Classification of thyroid ultrasound videos}{Sept. 2025 - present}
%     \item Leverage image foundation models to create a multiple video malignancy classification model
%     % \item Novel integration of ultrasound videos with radiological reports with application to improved cancer risk stratification
% \end{projectlong}

\begin{projectlong}{Robust cytological image representation learning}{Feb. 2025 - present}
    \item Developed the first cytology foundation model using self-supervised learning 
    \item Curated the largest cytology dataset of 1.4 million images using public and private datasets
    \item Outperformed natural image and histopathology foundation models on 2 out of 3 tasks
\end{projectlong}

\begin{projectlong}{Wearable device adherence in heart failure patients}{Feb. 2023 - Sept. 2024}
    \item Led data processing and statistical analysis for a randomized controlled trial 
    \item Evaluated digital monitoring adherence in heart failure patients
    \item Identified signal patterns and adherence trends from Fitbit and symptom-tracking survey data
\end{projectlong}

% \begin{projectlong}{A Fitbit foundation model}{Jan. 2025 - present}
%     \item Developing a transformer based Fitbit foundation model using ~36k participants from the All of Us Database
%     \item Downstream applications to studies with small sample sizes using Fitbit devices, such as our private heart failure dataset
% \end{projectlong}

% \begin{tabularx}{\linewidth}{ @{}l r@{} }
% \textbf{Some Project} & \hfill \href{https://some-link.com}{Link to Demo} \\[3.75pt]
% \multicolumn{2}{@{}X@{}}{long long line of blah blah that will wrap when the table fills the column width long long line of blah blah that will wrap when the table fills the column width long long line of blah blah that will wrap when the table fills the column width long long line of blah blah that will wrap when the table fills the column width}  \\
% \end{tabularx}



% %----------------------------------------------------------------------------------------
% %	PATENTS
% %----------------------------------------------------------------------------------------
% \section{Patents}
% \begin{refsection}[patents.bib]
% \nocite{*}
% \printbibliography[heading=none]
% \end{refsection}

% % \section{Patents}
% % % apply local override for the *next* bibliography only
% % \AtNextBibliography{%
% %   \setcounter{maxnames}{8}%
% %   \setcounter{minnames}{8}%
% % }
% % \nocite{*} % if you want to include all patent entries
% % \printbibliography[heading=none, keyword=patent]

\enlargethispage{3\baselineskip}

%----------------------------------------------------------------------------------------
%	SKILLS
%----------------------------------------------------------------------------------------
\section{Skills}
\begin{tabularx}{\linewidth}{@{}l X@{}}
Deep learning & \normalsize{Self-supervised training, contrastive learning, multi-modal learning, transformers, CNNs}\\
Technical &  \normalsize{Python, Pytorch, pandas, SQL, C, numpy, Scipy, scikit-learn, git, openCV, OpenSlide, Hugging Face, Weights \& Biases} 
\end{tabularx}
% \begin{tabularx}{\linewidth}{@{}l X@{}}
% Some Skills &  \normalsize{This, That, Some of this and that etc.}\\
% Some More Skills  &  \normalsize{Also some more of this, Some more that, And some of this and that etc.}\\  
% \end{tabularx}


% \vfill
\center{\footnotesize Last updated: \today}

\end{document}
